lib/modules/apostrophe-modal/public/js/modal.js

```diff
@@ -535,60 +534,6 @@ apos.define('apostrophe-modal', {
       }
     };
 
-    // Calculates the appropriate modal body height by subtracting
-    // header, breadcrumb, and footer heights and an additional
-    // 50 pixels from the browser window height. Invoked for you
-    // by `self.show()`.
```

Comment:
There is a lot of hard-won logic here. How sure are we that this still works for all the situations it was meant to address?

Reply:
It sounds like the answer is "we fixed it with flexbox," however A2 supports editing in IE11. Is this tested in IE11?

Reply:
The only purpose of the function is to compute and set height for the modal and its header / contents / footer. Flexboxes allow to make it easily, dynamically with less ressources. You right we sould take car with this type of modifications but I took the liberty to do it because the modal was already buggy on my moderns chrome and firefox, I had two superimposed scroll which was really uncomfortable as a user.
Also I checked on IE11 with BrowserStack, the height of the modal seems ok,  the modal's footer is broken and is out of the modal at the bottom of the window, but this bug is present in the main branch too.

Reply:
Just fixed the modal's footer in IE11, but there are still weirds rendering on the editor modal (which was already existing)